src/cli/mod.rs

```diff
@@ -65,7 +66,8 @@ impl Default for Channel {
 pub enum Cmd {
     /// View information about types
     Types(types::Cmd),
-    /// Guess the XDR type
+    /// Guesses the XDR type. All possible types are be printed for given XDR using given output format
+    /// (list views with each item on a new line by default).
```

Comment:
I suggest we format this slightly different.
```suggestion
    /// Guesses the XDR type.
    ///
    /// Prints a list of types that the XDR values can be decoded into.
```